### PR TITLE
Relax padrino-core dependency for padrino v0.14

### PR DESCRIPTION
**Pull request by sue445**
latest `padrino` is v0.14.0.1, but `rspec-padrino` is locking with v0.13.x
So I relaxed dependency

It seems no problem at my environment. (padrino v0.14.0.1)

### CODE REVIEW

File: rspec-padrino.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "sinatra", ">= 0"
-  spec.add_runtime_dependency "padrino-core", "~> 0.13.0"
+  spec.add_runtime_dependency "padrino-core", "~> 0.13"
```
Comment (udzura) [inline rspec-padrino.gemspec]: Good and I prefer `>= 0.13`

Comment (sue445) [inline rspec-padrino.gemspec]: I fixed and squash push at 4d3a8095a48f1006c024574f36ed5d70b54e4c21

**Comment by udzura** [comment]
Looks OK to me! How about you? @namusyaka

**Comment by namusyaka** [comment]
Looks good.

@udzura Btw, I really sorry, I'm unlikely to be able to work for a while because I have lost my physical condition.
I would like to entrust you release next version.

**Pull request merged**